content/en/cosign/hardware-based-tokens.md

```diff
@@ -95,40 +102,40 @@ You can then use the normal `cosign` commands to sign images and blobs with your
 **NOTE**: The default PIN is `123456`.
 
 ```shell
-$ cosign sign --sk gcr.io/user-vmtest2/demo
+$ cosign sign --sk $IMAGE_DIGEST
 Enter PIN for security key:
 Please tap security key...
-Pushing signature to: gcr.io/user-vmtest2/demo:sha256-410a07f17151ffffb513f942a01748dfdb921de915ea6427d61d60b0357c1dcd.sig
+Pushing signature to: gcr.io/dlorenc-vmtest2/demo:sha256-410a07f17151ffffb513f942a01748dfdb921de915ea6427d61d60b0357c1dcd.sig
```

Comment:
Didn't we decide to make thinks more generic / not tied to a user?

Reply:
We did, although do we want to point folks to a test image that we host somewhere? Or are we expecting users to have their own images on hand?